spec/features/timeout_spec.rb

```diff
@@ -2,9 +2,10 @@
 
 RSpec.feature "Teacher Student Loan Repayments claims", js: true do
   let(:one_second_in_minutes) { 1 / 60.to_f }
+  let(:two_second_in_minutes) { 2 / 60.to_f }
```

Comment:
Can I be really annoying and ask for proper pluralisation on this? 😄

Reply:
done